src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -120,8 +125,8 @@ def update_stats(self):
 
                 # Add supported counters to MIBs list and store counters values
                 for (counter, counter_type), counter_mib_id in CounterMap.items():
-                    # Only ingress queues are supported
-                    mib_oid = (if_index, int(DirectionTypes.INGRESS), queue + 1, counter_mib_id)
+                    # Only egress queues are supported
```

Comment:
>egress [](start = 27, length = 6)

What about ingress? Add some comment?

Reply:
There is appropriate comment already: "# Only egress queues are supported"
Which information do we need to extend?

Reply:
So why 'only egress'?

---
In reply to: [158642682](https://github.com/andrii-savka/sonic-snmpagent/pull/5#discussion_r158642682) [](ancestors = 158642682)

Reply:
As far as I know, SONiC does not support ingress queues, only egress.